deploy/buildtools/bitbuilder.py

```diff
@@ -505,7 +509,7 @@ def on_build_failure():
             return False
 
         hwdb_entry_name = self.build_config.name
-        xclbin_path = cl_dir + "/bitstream/build_dir.xilinx_u250_gen3x16_xdma_3_1_202020_1/firesim.xclbin"
+        xclbin_path = cl_dir + f"/bitstream/build_dir.{self.device}/firesim.xclbin"
```

Comment:
Maybe comment that the file path structure (i.e. `build_dir` and `firesim.xclbin`) is determined by the `Makefile`

Reply:
yeah either that or something like `to-azure` which would be consistent with the `to-aws` directory that has the final routed dcp file.